src/Store.js

```diff
@@ -75,13 +75,7 @@ class Store {
    * @return {Array} array of ArtemisData records
    */
   addModels = (type, data) => {
-    let records = []
-
-    transaction(() => {
-      records = data.map(obj => this.addModel(type, obj))
-    })
-
-    return records
+    return transaction(() => data.map((obj) => this.addModel(type, obj)))
```

Comment:
refactored this pattern in a few places. `transaction` will return whatever the action fn returns, so we don't need the extra var.